OpenSim.xcodeproj/project.pbxproj

```diff
@@ -385,7 +385,7 @@
 				INFOPLIST_FILE = OpenSim/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks";
 				MACOSX_DEPLOYMENT_TARGET = 10.10;
-				PRODUCT_BUNDLE_IDENTIFIER = "com.pop-tap.OpenSim";
+				PRODUCT_BUNDLE_IDENTIFIER = com.bazscott.mac.OpenSim;
```

Comment:
@johndpope Probably wouldn't commit this change, I just did this for my local copy :)

Reply:
I manually edited using github editor. should be good to go https://github.com/luosheng/OpenSim/pull/44/commits/c00d7cb44f9f3963394052dcef68c0bff03bfcc8